docs/painless/painless-contexts/painless-field-context.asciidoc

```diff
@@ -26,3 +26,64 @@ a customized value for each document in the results of a query.
 *API*
 
 The standard <<painless-api-reference, Painless API>> is available.
+
+
+*Example*
+
+To run this example, first follow the steps in
+<<painless-context-examples, context examples>>.
+
+We will use two scripts to create two additional fields for each search hit.
+These script fields will output computed custom information.
+
```

Comment:
In general, I recommend sticking with the present tense and avoiding "we" except for cases where "we" means "Elastic" (particularly in cases where you're making a recommendation). In this case, I'd probably edit this to read:

You can then use these example scripts to compute custom information and output it to two new fields.